site/app/views/grading/ElectronicGraderView.php

```diff
@@ -832,6 +832,10 @@ public function detailsPage(Gradeable $gradeable, $graded_gradeables, $teamless_
         elseif (count($grader_registration_sections) === 0) {
             $message = 'Notice: You are not assigned to grade any students for this gradeable.';
         }
+        if ($inquiry_status) {
+            $message .= ($message === "") ? "Notice: You are viewing students with active grade inquiry." : "\nNotice: You are viewing students with active grade inquiry.";
```

Comment:
I'm confused about what's going on here.  If you are trying to conditionally add a newline, can't you just do that rather than duplicating the same string?

Also, this is not grammatically correct.